src/modules/configuration/src/lib/Configuration.c

```diff
@@ -90,6 +90,7 @@ static char* LoadConfigurationFromFile(const char* fileName)
         g_iotHubManagementEnabled = IsIotHubManagementEnabledInJsonConfig(jsonConfiguration);
         g_iotHubProtocol = GetIotHubProtocolFromJsonConfig(jsonConfiguration, ConfigurationGetLog());
         g_gitManagementEnabled = GetGitManagementFromJsonConfig(jsonConfiguration, ConfigurationGetLog());
+        FREE_MEMORY(g_gitBranch);

```

Comment:
>FREE_MEMORY(g_gitBranch); [](http://example.com/codeflow?start=8&length=25)

Add one empty line as separator before the FREE_MEMORY line to split this into two distinct blocks.